Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusNotificationBus.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+
+namespace AzToolsFramework::Prefab
+{
+    class PrefabFocusNotifications : public AZ::EBusTraits
```

Comment:
nit: An API comment for this would likely be useful

Reply:
question: Are we sure we want this to be a global bus? Is there another Id we could use (Scene or EntityContextId or even viewport potentially?)

Reply:
Adding API comments, and made the bus addressed on EntityContextId.

Reply:
Nice one thanks!